shimmingtoolbox/prepare_fieldmap.py

```diff
@@ -101,3 +116,46 @@ def prepare_fieldmap(list_nii_phase, echo_times, unwrapper='prelude', mag=None,
 
     # return fieldmap_hz_gaussian
     return fieldmap_hz
+
+
+def correct_2pi_offset(unwrapped, mag, mask, validity_threshold):
+    """ Removes 2*pi offsets from unwrapped for a time series. If there is not offset, it returns the same array.
```

Comment:
It should be "unwrapper" or "unwrapping" right ?

Reply:
Also "If there is no offset"

Reply:
Corrected here: ce1d691